packages/accessibility/src/behaviors/Carousel/carouselItemBehavior.ts

```diff
@@ -16,7 +17,11 @@ const carouselItemBehavior: Accessibility<CarouselItemProps> = props => ({
   },
 
   keyActions: {
-    root: {},
+    root: {
+      arrowKeysNavigationStopPropagation: {
```

Comment:
Let's add a unit test for this.

It should check that left/right arrows sent to a child of the container (the visible item for instance, or a child of that visible item) should not move the slides.

Reply:
thanks :) for this catch. Tests added,  as well a specification line in the top of file.